data-protocols/ids/ids-api-configuration/src/main/java/org/eclipse/edc/protocol/ids/api/configuration/IdsApiSettings.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.protocol.ids.api.configuration;
+
+import org.eclipse.edc.runtime.metamodel.annotation.Setting;
+import org.eclipse.edc.web.spi.configuration.WebServiceSettings;
+
+public class IdsApiSettings implements WebServiceSettings {
+
+    public static final String IDS_API_CONFIG = "web.http.ids";
+    public static final String IDS_API_CONTEXT_ALIAS = "ids";
+
+    public static final int DEFAULT_IDS_PORT = 8282;
+    public static final String DEFAULT_IDS_API_PATH = "/api/v1/ids";
+
+    @Setting
+    public static final String IDS_WEBHOOK_ADDRESS = "ids.webhook.address";
+    public static final String DEFAULT_IDS_WEBHOOK_ADDRESS = "http://localhost";
+
```

Comment:
These two constants would fit better in the `Extension` class, as they are used only there